plugins/colorschemes/base16/default.nix

```diff
@@ -0,0 +1,118 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+# We configure this plugin manually (no `settings` option) so there is no point in using
+# `mkNeovimPlugin` here.
+  helpers.vim-plugin.mkVimPlugin config {
+    name = "base16";
+    isColorscheme = true;
+    originalName = "base16.nvim";
+    defaultPackage = pkgs.vimPlugins.base16-nvim;
+
+    maintainers = [maintainers.GaetanLepage];
+
+    imports = [
+      (
+        mkRemovedOptionModule
+        ["colorschemes" "base16" "customColorScheme"]
+        "You can directly provide an attrs to the `colorschemes.base16.colorscheme` option."
+      )
+    ];
+
+    extraOptions = {
+      useTruecolor = mkOption {
```

Comment:
This option seems pretty useless as it simply maps to a builtin neovim option. Shouldn't we deprecate it with a mkRenameOption to the correct neovim builtin option?

Reply:
Sure.